### PR TITLE
Utkarsha/ts migration constant files in accounts package

### PR DESCRIPTION
**Pull request by utkarsha-deriv**
## Changes:

Please provide a summary of the change.

### Screenshots:

Please provide some screenshots of the change.

### CODE REVIEW

**Review by shaheer-deriv — APPROVED** [review]

File: packages/account/src/Constants/onfido-phrases.tsx [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Localize } from '@deriv/translations';
+import { TOnfidoPhrases } from '../Types/onfido-phrases.type';
 
-const getOnfidoPhrases = () => ({
+const getOnfidoPhrases = (): TOnfidoPhrases => ({
```
Comment (yauheni-deriv) [inline packages/account/src/Constants/onfido-phrases.tsx]:
```suggestion
const getOnfidoPhrases = (): LocaleConfig['phrases'] => ({
```

File: packages/account/src/Constants/onfido-phrases.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import { Localize } from '@deriv/translations';
+import { TOnfidoPhrases } from '../Types/onfido-phrases.type';
```
Comment (yauheni-deriv) [inline packages/account/src/Constants/onfido-phrases.tsx]:
```suggestion
import { LocaleConfig } from 'onfido-sdk-ui';
```

File: packages/account/src/Constants/onfido-phrases.tsx (hunk 1 above)
Comment (yauheni-deriv) [reply packages/account/src/Constants/onfido-phrases.tsx]: do we really need to specify exactly what string we are passing to onfido?

File: packages/account/src/Types/onfido-phrases.type.ts [hunk 3]
```diff
@@ -0,0 +1,313 @@
+export type TOnfidoPhrases = {
```
Comment (yauheni-deriv) [inline packages/account/src/Types/onfido-phrases.type.ts]: maybe we can simply it and just use 
`import { LocaleConfig } from 'onfido-sdk-ui';`
how do you think?

File: packages/account/src/Constants/routes-config.ts [hunk 4]
```diff
@@ -20,13 +20,20 @@ import {
     AccountClosed,
     DeactivateAccount,
     LanguageSettings,
-} from 'Sections';
+} from '../Sections';
```
Comment (amina-deriv) [inline packages/account/src/Constants/routes-config.ts:23]: why do we need this change?

File: packages/account/src/Types/onfido-phrases.type.ts (hunk 3 above)
Comment (utkarsha-deriv) [reply packages/account/src/Types/onfido-phrases.type.ts]: Yes, I think that would be better. Thank you

File: packages/account/src/Constants/routes-config.ts (hunk 4 above)
Comment (utkarsha-deriv) [reply packages/account/src/Constants/routes-config.ts:23]: For some reason the editor throws error saying `Cannot find module .....` relative path resolved the issue

File: packages/account/src/Constants/routes-config.ts [hunk 5]
```diff
@@ -20,13 +20,20 @@ import {
...
+    is_appstore: boolean;
+};
 
 // Error Routes
 const Page404 = React.lazy(() => moduleLoader(() => import(/* webpackChunkName: "404" */ 'Modules/Page404')));
+export type TPage404 = typeof Page404;
 
 // Order matters
-const initRoutesConfig = ({ is_appstore }) => [
+const initRoutesConfig = ({ is_appstore }: TInitRoutesConfig): TRouteConfig[] => [
```
Comment (likhith-deriv) [inline packages/account/src/Constants/routes-config.ts]: May be u can remove `is_appstore`?

**Review by yauheni-deriv — APPROVED** [review]

File: packages/account/src/Components/Routes/__tests__/binary-link.spec.tsx [hunk 6]
```diff
@@ -49,13 +44,9 @@ describe('<BinaryLink />', () => {
 
         expect(() =>
             render(
```
Comment (likhith-deriv) [inline packages/account/src/Components/Routes/__tests__/binary-link.spec.tsx:45]: **NIT:** This render could be written as a function and then re-used in the test suites

**Review by likhith-deriv — APPROVED** [review]

**Review by amina-deriv — APPROVED** [review]

**Comment by amina-deriv** [comment]
LGTM 👌

**Review by suisin-deriv — APPROVED** [review]

**Review by matin-deriv — APPROVED** [review]
Nicely done

**Pull request merged**